.github/workflows/build.yml

```diff
@@ -11,9 +11,9 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - name: centos-stream-8
-            shortcut: cs8
-            container-name: el8stream
+          - name: centos-stream-10
+            shortcut: cs10
+            container-name: el10stream
```

Comment:
Sorry a bit nitpicking, but can you add cs10 after cs9 ? :)

Reply:
Sure, no problem.